.circleci/continue_config.yml

```diff
@@ -131,6 +131,7 @@ jobs:
             - run:
                 name: 'Run Unit Tests'
                 command: |
+                  export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib:/usr/local/lib64"
```

Comment:
Another option is to update the dockerfile (to make sure the paths are picked up like we do for the runtime image) for the dependency image and rebuild it.

Reply:
I think we should also install the dependencies in a custom directory using `INSTALL_PREFIX` and use the ldconfig approach.
Let's merge this and use a follow-up PR?

Reply:
Sure.

Reply:
We can also avoid this issue by linking proxygen statically. The proxygen CMake recommends that.